src/Plugin.php

```diff
@@ -70,9 +70,11 @@ public static function post_gallery($output = '', $atts) {
     $atts['post_type'] = 'attachment';
     $atts['post__in'] = $atts['include'];
     $layout = !empty($atts['layout']) ? $atts['layout'] : apply_filters('gallerya/default_layout', self::DEFAULT_LAYOUT);
+    $min_images_to_show_nav = apply_filters('gallerya/min_images_to_show_nav', 5);
```

Comment:
1. Agree, me too don't like `min_images_to_show_nav `. So I'll check first check Flickity and, if nothing is in there, going with `nav_count_min`. 
2. Nope, mistake after changed operator from `>` to `>=`. I'll modify it to `6`. Thanks. 👍